src/Faker/Provider/ne_NP/Address.php

```diff
@@ -62,33 +62,7 @@ class Address extends \Faker\Provider\Address
         'Yala', 'Yalbang', 'Yamphodin', 'Yrsa',
     );
 
-    protected static $country = array(
```

Comment:
Why? This formatter is used to generate a random country name. A single country isn't random.